cpp/client/ct.cc

```diff
@@ -981,10 +981,8 @@ int Monitor() {
 // Exit code upon abnormal exit (CHECK failures): != 0
 // (on UNIX, 134 is expected)
 int main(int argc, char** argv) {
+  util::InitCT(&argc, &argv);
```

Comment:
Pretty sure you have to set the usage message *before* calling `google::ParseCommandLineFlags`?